test/inspector/test-inspector.js

```diff
@@ -199,6 +199,36 @@ function testI18NCharacters(session) {
   ]);
 }
 
+function testCommandLineAPI(session) {
+  session.sendInspectorCommands([
+    [ // we can use `require` outside of a callframe with require in scope
+      {
+        'method': 'Runtime.evaluate', 'params': {
+          'expression': 'typeof require("fs").readFile === "function"',
+          'includeCommandLineAPI': true
+        }
+      }, (message) => assert.strictEqual(true, message['result']['value'])
+    ],
+    [ // the global require does not have require.cache
+      {
+        'method': 'Runtime.evaluate', 'params': {
+          'expression': 'require.cache === undefined',
+          'includeCommandLineAPI': true
+        }
+      }, (message) => assert.strictEqual(true, message['result']['value'])
+    ],
+    [ // the `require` in the module shadows the command line API's `require`
+      {
+        'method': 'Debugger.evaluateOnCallFrame', 'params': {
+          'callFrameId': '{"ordinal":0,"injectedScriptId":1}',
+          'expression': 'typeof require.cache',
+          'includeCommandLineAPI': true
+        }
+      }, (message) => assert.strictEqual('object', message['result']['value'])
+    ],
+  ]);
+}
```

Comment:
Can you check that:

1. `require(name)` twice returns the same object, and
2. that `delete require.cache[require.resolve(name)]` followed by require again returns a new object, and
3. that calling require again returns the same object?

`name` could be e.g. `const name = path.join(common.fixturesDir, 'empty');`